binding/SkiaSharp.Skottie/Animation.cs

```diff
@@ -56,8 +60,8 @@ public static bool TryCreate (SKStream stream, [System.Diagnostics.CodeAnalysis.
 		{
 			_ = stream ?? throw new ArgumentNullException (nameof (stream));
 
-			animation = GetObject (SkottieApi.skottie_animation_make_from_stream (stream.Handle));
-			return animation != null;
+			using var data = SKData.Create (stream);
+			return TryCreate (data, out animation);
```

Comment:
All animations are now created through the SKData overload, which we then use to detect the BOM.

We _could_ use the stream, but then we have to use buffering and all other manner of complex mechanisms. This way we load the full data, and then optionally skip 4 bytes before passing it to C++.